Intersect.Client/Core/Graphics.cs

```diff
@@ -364,11 +364,7 @@ public static void DrawInGame()
                         entity.DrawName(null);
                         if (entity.GetType() != typeof(Event))
```

Comment:
`!(entity is Event)`

Reply:
i didn't touched that one, but sure, this looks more clean, done 👍🏽 !